.github/workflows/deploy.yml

```
@@ -135,7 +140,7 @@ jobs:
       - name: Setup workspace
         env:
           HUSKY: 0
-          REMOTE_MODULES_BASE_PATH: 'https://cdn.rudderlabs.com/${{ inputs.s3_dir_path }}/modern/plugins'
+          REMOTE_MODULES_BASE_PATH: '${{ inputs.cdn_url_host }}/${{ inputs.s3_dir_path }}/modern/plugins'
```

actionlint (1.7.7)

property "cdn_url_host" is not defined in object type {base_cdn_url: string; bugsnag_release_stage: string; environment: string; monorepo_release_version: string; release_ticket_id: string; s3_dir_path: string; s3_dir_path_legacy: string; trigger_source: string; use_pr_head_sha: bool; version_suffix: string}

(expression)